components/storage/blockdevice/COMPONENT_SD/SDBlockDevice.cpp

```diff
@@ -1129,29 +1129,14 @@ void SDBlockDevice::_spi_wait(uint8_t count)
 
 void SDBlockDevice::_spi_init()
 {
-    _spi.lock();
+    _spi.select();
```

Comment:
This is actually asserting SSEL, which the original code didn't. Is that valid? During this stage the card isn't speaking SPI yet, so I'm not sure how it will interpret the SSEL signal. I think I'd be paranoid and leave it as just lock/unlock. The original `cs = 1` seemed to be redundant - the chip should already have been deselected on entry.

Reply:
Done